Draft

TinaL5 opened the pull request:
Please let me know if you have feedback about the coding, visualizations, or anything else. Descriptions and tasks have not yet been added; the focus for the anonymized event data will be interpretation, and the focus for the Champions League data will be creation.

TinaL5 committed:
Draft 

Please let me know if you have feedback about the coding, viz, or anything else. Descriptions and tasks have not yet been added.

byrcyb commented:
Some initial comments:
- remove `from pandas import read_csv` as you alias pandas as pd above, and then when you call `read_csv` below you use `pd.read_csv("")`
- I would add brief (1 sentence) markdown descriptions to explain what the (below) code cell is doing and why (the context). E.g. this code will count all the different types of actions you can take on a soccer pitch and sort it by frequency. 
- For the first graph, you should explain what Start Y and Start X refer to, as it's not clear that refers to cartesian coordinates on the soccer pitch.
-  The timeline graphs are really nice. I would also explain that the whole soccer game is 5,400 seconds (0 to 5,400) i.e. 90 minutes x 60 sec/min.
- What's the purpose of showing the "home team passes, second half" graph? Explain the context.
- Remove the 2nd `import pandas as pd`
- Make sure you have labels on all your axes in graphs e.g. the Bayern: Type of Goal
- Explain what "Against woodwork" means
- I really like the direction of the notebook especially the last 2 graphs.
- Perhaps combine all the `.min(), .max(), .median(), .mean()` lines into 1 code cell and print it out in one block
- I would wrap things up with a conclusion. I wouldn't add too much more as the notebook is getting a bit long.

byrcyb commented:
Couple of other comments, 
- Please move your notebook to a separate folder.
- Add Callysto banners to top and bottom of notebook.

byrcyb commented:
@TinaL5 I also noticed there's an extra notebook hanging out in the notebooks folder, can you delete this file `notebooks/Cleanup.ipynb`as part of your next set of commits?

TinaL5 committed:
Delete Cleanup.ipynb

TinaL5 committed:
Descriptions for viz

TinaL5 committed:
Delete soccer-student.ipynb

TinaL5 committed:
DRAFT Addition of interactive; Updates to wording

Commit:
Split notebook into three parts

The full notebook is quite long for a student to complete. Split content into three notebooks.

Commit:
reword X Y axis, refactor min, max, etc

lgfunderburk reviewed (approved):
Code, plots and summaries look great!
Refactored code into three notebooks as the full notebook was quite long.
